src/browser/api/window.js

```diff
@@ -1697,13 +1697,21 @@ Window.defineDraggableArea = function() {};
 
 Window.updateOptions = function(identity, updateObj) {
     let browserWindow = getElectronBrowserWindow(identity, 'update settings for');
+    let { uuid, name } = identity;
+    let diff = [];
 
     try {
         for (var opt in updateObj) {
             if (optionSetters[opt]) {
+                let optionValues = { optionName: opt };
+                optionValues.oldVal = getOptFromBrowserWin(opt, browserWindow);
                 optionSetters[opt](updateObj[opt], browserWindow);
+                optionValues.newVal = getOptFromBrowserWin(opt, browserWindow);
```

Comment:
I think some options might be objects (see cornerRounding - https://cdn.openfin.co/docs/javascript/alpha/Window.html).  Do we need to clone/copy the oldVal/newVal here?

Reply:
Pushed a fix.